### DISCUSSION TITLE
Editing one cell to cause another to update.

### DISCUSSION THREAD

**Discussion by ethanfox** [bryanmylee/svelte-headless-table#126] [Q&A]
I am creating a data grid where designers specify products. Two columns in the table are "sales rep" and "source". I want to be able to edit the sales rep column(accomplished) and when that changes have the source column be updated.(what I am having trouble with) . Is it possible to initiate a rerender of a cell from a different cell in this library?

Picture of row:
<img width="1384" alt="Screenshot 2023-04-15 at 7 16 43 PM" src="https://user-images.githubusercontent.com/22962600/232259534-3524414f-5dcc-41e4-9906-a103e7567fb7.png">

**Comment by nicholascostadev** [comment]
Hi @ethanfox, did you manage to make the column rerender? I'm having the same issue right now.

**Comment by ethanfox** [comment]
Unfortunately I was not. I ended up making my own data grid component.
It was pretty easy to set up. Had to us copilot to plow through some of the logic needed. I can show you the code if you want.
